qa/workunits/cephtool/test.sh

```diff
@@ -1846,6 +1846,12 @@ function test_mon_cephdf_commands()
   expect_false test $cal_raw_used_size != $raw_used_size
 }
 
+function test_mon_stdin_stdout()
```

Comment:
Right - the "standard" (if there is one) is to not add signed-off-by for cherry-picks, since they do not involve any original work. But if someone insists, I don't make an issue out of it.